src/App/FreeCADInit.py

```diff
@@ -807,6 +809,7 @@ def report(self,msg,func,*args,**kargs):
 App.Units.ElectricalInductance  = App.Units.Unit(2,1,-2,-2)
 App.Units.ElectricalConductance = App.Units.Unit(-2,-1,3,2)
 App.Units.ElectricalResistance  = App.Units.Unit(2,1,-3,-2)
+App.Units.ElectricalConductivity = App.Units.Unit(-3,-1,3,2)

```

Comment:
According to https://de.wikipedia.org/wiki/Siemens_(Einheit) this unit is wrong. It must be (-2,-1,3,2)